packages/jellyfish-api-core/__tests__/category/blockchain/getMempoolInfo.test.ts

```diff
@@ -0,0 +1,30 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { BigNumber } from '../../../src'
+
+describe('getMempoolInfo', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should getMempoolInfo', async () => {
+    const mempoolInfo = await client.blockchain.getMempoolInfo()
+
+    expect(typeof mempoolInfo.loaded).toStrictEqual('boolean')
+    expect(mempoolInfo.size).toBeInstanceOf(BigNumber)
+    expect(mempoolInfo.bytes).toBeInstanceOf(BigNumber)
+    expect(mempoolInfo.usage).toBeInstanceOf(BigNumber)
+    expect(mempoolInfo.maxmempool).toBeInstanceOf(BigNumber)
+    expect(mempoolInfo.mempoolminfee).toBeInstanceOf(BigNumber)
+    expect(mempoolInfo.minrelaytxfee).toBeInstanceOf(BigNumber)
```

Comment:
this `assert` alerted me
can you declare `mempoolInfo` with type??
```
const mempoolInfo: MempoolInfo = ...
```

Reply:
Added the type annotation.

Reply:
I actually expect you will get error after specific the type but nope
TBH, this assertion is not **type safe**
`expect(mempoolInfo.size).toBeInstanceOf(BigNumber)`
please change to 
`expect(mempoolInfo.size instanceof BigNumber).toStrictEqual(true)`

Reply:
`instanceof` expects the left hand value to be of `any` or object type.

```
expect(mempoolInfo.size instanceof BigNumber).toStrictEqual(true)

    packages/jellyfish-api-core/__tests__/category/blockchain/getMempoolInfo.test.ts:25:12 - error TS2358: The left-hand side of an 'instanceof' expression must be of type 'any', an object type or a type parameter.
```

We could do this only if we don't type annotate `const mempoolInfo`

```
// We can't do instanceof if we do this
const mempool: MempoolInfo = await client.blockchain.getMempoolInfo()

expect(mempool.size instanceof BigNumber).toStrictEqual(true)
```

```
// we can do this
const mempool = await client.blockchain.getMempoolInfo()

expect(mempool.size instanceof BigNumber).toStrictEqual(true)
```

Reply:
that's the safety way to test
you gonna update the interface
https://github.com/DeFiCh/jellyfish/pull/364/files#r648974024

Reply:
Some context for reference 

https://github.com/DeFiCh/jellyfish/issues/18
https://github.com/DeFiCh/jellyfish/pull/47
https://github.com/DeFiCh/jellyfish/issues/93
https://github.com/DeFiCh/jellyfish/issues/104
https://github.com/DeFiCh/jellyfish/pull/128

Reply:
Made the necessary changes and updated the docs.